internal/server/option/auth_test.go

```diff
@@ -308,7 +308,8 @@ func TestAuthenticate(t *testing.T) {
 			authInterceptor: &authInterceptor{
 				cfg: config.ServerConfig{
 					OIDCConfig: &libOIDC.Config{
-						IssuerURL: testIDPIssuer,
+						IssuerURL:     testIDPIssuer,
+						UsernameClaim: "oidc-username",
```

Comment:
Nit: Instead of `oidc-username` I would probably use a real claim name like `preferred_username` or a nonsense word like `foo`, only because I think it's too easy to look at `oidc-username` and be fooled into thinking this value _means something_. I'd probably say the same for the claim's value down on line 326.

```suggestion
						UsernameClaim: "oidc-username",
```